packages/next/taskfile.js

```diff
@@ -228,6 +228,7 @@ export async function copy_vercel_og(task, opts) {
     join(__dirname, 'src/compiled/@vercel/og/package.json'),
     {
       name: '@vercel/og',
+      version: require('@vercel/og/package.json').version,
```

Comment:
Could we copythe whole package.json instead of selectively choosing properties?

Reply:
The folder structure is changed, so we need to re-map the files, also filtered unused stuff out